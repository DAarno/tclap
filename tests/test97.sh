#!/bin/bash
./simple-test.sh `basename $0 .sh` test32 -b 5 -s foo
