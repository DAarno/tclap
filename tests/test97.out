PARSE ERROR: Argument: -b AND -s provided.
             Only one is allowed.

Brief USAGE: 
   ../examples/test32  [-h] [--version] {-a <int>|-b <int>|-s <str>}

For complete USAGE and HELP type: 
   ../examples/test32 --help

